etl/steps/data/garden/hmd/2023-09-19/hmd.meta.yml

```diff
@@ -14,12 +25,13 @@ tables:
     variables:
       central_death_rate:
         title: Central death rate
-        description_short: Central death rate between ages x and x+n.
-        unit: deaths per people
+        description_short: |-
+          For a given population or cohort, the central death rate at age x during a given period of 12 months is found by dividing the number of people who died during this period while aged x by the average number who were living in that age group during the period.
+        unit: deaths per 1,000 people
       probability_of_death:
         title: Probability of death
         unit: ""
```

Comment:
Could multiply by 100 and present as a %, but up to you.

Reply:
Totally, I'll scale this